tez.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: tez
+  version: 0.10.4
+  epoch: 0
+  description: Apache Tez
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - openjdk-8
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - ca-certificates-bundle
+      - maven
+      - openjdk-8
+      - wolfi-base
+      - wolfi-baselayout
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/java-1.8-openjdk
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/apache/tez
+      tag: rel/release-${{package.version}}
+      expected-commit: 5b5fff619683a204d0c62f76ddab50e3ddc38760
+
+  - uses: patch
+    with:
+      patches: CVE-2024-7254.patch
```

Comment:
Please note that while it patches the `com.google.protobuf.protobuf-java` version from 3.24.4 to 3.25.5, the CVE is still not fixed as there's still a protobuf Hadoop-shaded depdendency.